src/plugins/ScryptEncrypter.test.ts

```diff
@@ -0,0 +1,34 @@
+import { ScryptEncrypter } from "./ScryptEncrypter";
+
+import { EncryptedPlaintextKey, KeyType } from "../types";
+
+test("encrypts and decrypts a key", async () => {
+  const key = {
+    type: KeyType.plaintextKey,
+    publicKey: "AVACYN",
+    privateKey: "ARCHANGEL",
+  };
+
+  const password = "This is a really cool password and is good";
+
+  const encryptedKey = await ScryptEncrypter.encryptKey({ key, password });
+
+  expect(encryptedKey).toBeTruthy();
+  expect(
+    (encryptedKey as EncryptedPlaintextKey).encryptedPrivateKey,
+  ).toBeTruthy();
+  expect(
+    (encryptedKey as EncryptedPlaintextKey).encryptedPrivateKey,
+  ).not.toEqual(key.privateKey);
```

Comment:
The test should verify that the encrypted string is exactly a certain value. This detects regressions where we accidentally change the encryption algorithm.